keras/layers/merge.py

```diff
@@ -18,6 +18,31 @@ def __init__(self, **kwargs):
     def _merge_function(self, inputs):
         raise NotImplementedError
 
+    def _compute_elemwise_op_output_shape(self, shape1, shape2):
+        """check if shapes of 2 tensors are compatible for element wise operations,
```

Comment:
For style purposes, the first line in the docstring should fit in one line (here you have 2), should be capitalized and end in a dot. This is automatically enforced in tf-keras, so we have to make the codebase compliant.